cdap-ui/app/cdap/components/SearchTextBox/SearchTextBox.scss

```diff
@@ -16,6 +16,7 @@
 
 .cask-search-textbox {
   position: relative;
+  padding: 0.5em;
```

Comment:
move all the styles to CSS in JS? @ajainarayanan

Reply:
You can use this for reference: https://github.com/cdapio/cdap/blob/develop/cdap-ui/app/cdap/components/AbstractWidget/RadioGroupWidget/index.tsx

Reply:
remove this changes because i am not used that component